packages/components/psammead-brand/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-brand",
-  "version": "0.3.4",
+  "version": "0.3.5",
```

Comment:
This should be either `0.4.0` or `1.0.0`

Reply:
It should be a 1. since it is a breaking change.